configuration/pipeline_config.json

```diff
@@ -40,6 +40,13 @@
         "3903ad8e-284e-48a0-881b-89c0b06b9334",
         "45939459-e4d5-4ad6-8650-3e7134b9a035"
       ]
+    },
+    {
+      "SourceType": "RecoveryCSVURLs",
+      "ContactsFileName": "imaqal_covid19_contacts",
```

Comment:
The version of pipeline_config in IMAQAL_COVID19 doesn't have a ContactsFileName option